appengine/typescript/README.md

```diff
@@ -0,0 +1,60 @@
+# App Engine TypeScript sample
+
+This sample provides an example of how to compile TypeScript files while
+deploying to App Engine.
+
+The `gcp-build` NPM script is used to trigger the TypeScript compilation
+process. This step happens automatically when deploying to App Engine, but must
+be performed manually when developing locally.
+
+## Setup
+
+Install dependencies:
+
+    With `npm`:
+
+        npm install
+
+    or with `yarn`:
```

Comment:
Personal preference, I would not include `yarn` here. Our Node docs say that you can use `npm` or `yarn`, no need to show everything twice here.

Reply:
Removed `yarn` for brevity.